build.gradle

```
@@ -15,6 +15,8 @@ dependencies {
     implementation 'com.google.code.gson:gson:2.8.8'
     implementation 'org.slf4j:slf4j-api:1.7.32'
     implementation 'ch.qos.logback:logback-classic:1.2.6'
+    implementation 'org.apache.logging.log4j:log4j-api:2.2.24.1'
+    implementation 'org.apache.logging.log4j:log4j-core:2.2.24.1'
```

A typo in version, please run the `gradle` tasks to verify dependencies after changing versions to verify. 

```suggestion
    implementation 'org.apache.logging.log4j:log4j-api:2.24.1'
    implementation 'org.apache.logging.log4j:log4j-core:2.24.1'
```

---

Good catch, thank you @navin772 !

---

Command to verify dependencies - `./gradlew dependencies` 
Check for any FAILED dependency errors.

---

> A typo in version, please run the `gradle` tasks to verify dependencies after changing versions to verify.

Hmm I assumed IntelliJ would warn if the dependency fetching fails. Resolved, build is successful now, thanks!